### PR TITLE
Add VAF and RDF computation

### PR DESCRIPTION
**Pull request by oerc0122**
Enable VAF and RDF computation following MD runs and independently through the Python interfaces.

Partially addresses #117.

### CODE REVIEW

File: janus_core/calculations/md.py [hunk 1]
```diff
@@ -18,11 +18,21 @@
...
 )
 from ase.md.verlet import VelocityVerlet
+
+try:
+    from ase.geometry.analysis import Analysis
+
+    ASE_GEOMETRY = True
+except ImportError:
+
+    ASE_GEOMETRY = False
```
Comment (ElliottKasoar) [inline janus_core/calculations/md.py:28]: Can we remove this with ASE 3.23?

File: janus_core/helpers/post_process.py [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+    vaf /= n_steps - np.arange(n_steps)
+
+    if fft:
+        vaf = np.fft.fft(vaf)
+
+    if filename is not None:
+        with open(filename, "w", encoding="utf-8") as out_file:
+            for elem in vaf:
+                print(elem, file=out_file)
```
Comment (ElliottKasoar) [inline janus_core/helpers/post_process.py:209]: We may want some kind of binary format option here, but maybe that can come later

File: janus_core/calculations/md.py (hunk 1 above)
Comment (alinelena) [reply janus_core/calculations/md.py:28]: yes shall be done

File: janus_core/calculations/md.py [hunk 3]
```diff
@@ -542,6 +560,76 @@ def _write_final_state(self) -> None:
...
+                self.post_process_kwargs.get("rdf_stop", len(data)),
+                self.post_process_kwargs.get("rdf_step", 1),
+            )
+
+            out_paths = [
+                self._build_filename(
+                    "rdf.dat", param_pref, str(ind), prefix_override=base_name
+                )
+                for ind in range(*slice_)
+            ]
```
[the remark this answers is not in the document]
Comment (alinelena) [reply janus_core/calculations/md.py:606]: yes but remember ase is in a not t space... we need the average.

File: janus_core/helpers/post_process.py (hunk 2 above)
Comment (alinelena) [reply janus_core/helpers/post_process.py]: probably we shall see h5md approach and use it... Jacob is already aware.

File: janus_core/calculations/md.py (hunk 1 above)
Comment (alinelena) [reply janus_core/calculations/md.py:29]: what about this?

File: janus_core/calculations/md.py [hunk 4]
```diff
@@ -315,7 +334,7 @@ def __init__(  # pylint: disable=too-many-arguments,too-many-locals,too-many-sta
 
         self.minimize_kwargs = minimize_kwargs if minimize_kwargs else {}
         self.restart_files = []
-        self.dyn = None
+        self.dyn: Union[Langevin, VelocityVerlet, ASE_NPT]
```
Comment (alinelena) [inline janus_core/calculations/md.py:337]: i think we have nph too

Comment (oerc0122) [reply janus_core/calculations/md.py:337]: `NPH` appears to be a derivative of `NPT`.

c.f. ll. 1137-1213

**Review by alinelena — DISMISSED** [review]

**Pull request merged**